reposerver/repository/repository.go

```diff
@@ -101,13 +103,16 @@ func NewService(metricsServer *metrics.MetricsServer, cache *reposervercache.Cac
 		repoLock:                  repoLock,
 		cache:                     cache,
 		metricsServer:             metricsServer,
-		newGitClient:              git.NewClient,
+		newGitClient:              git.NewClientExt,
```

Comment:
Why is this change necessary?

Reply:
Switching to `NewClientExt` method here because it has `rootPath` parameter.